packages/ir-sdk/fern/apis/ir-types-latest/definition/dynamic/types.yml

```diff
@@ -0,0 +1,107 @@
+imports:
+  commons: ../commons.yml
+  declaration: ./declaration.yml
+  root: ../types.yml
+types:
+  NamedParameter:
+    properties:
+      name: commons.NameAndWireValue
+      typeReference: TypeReference
+
+  NamedType:
+    docs: |
+      Represents the type of a parameter that can be used to generate a dynamic type.
+    union:
+      alias: AliasType
+      enum: EnumType
+      object: ObjectType
+      discriminatedUnion: DiscriminatedUnionType
+      undiscriminatedUnion: UndiscriminatedUnionType
```

Comment:
just a note for the future that we'll want to eventually add an `optional<boolean>` inline